src/resources/js/ian-client.js

```diff
@@ -11,6 +11,13 @@
 		Ian.consent = Ian.notifications.dataset.consent;
 		Ian.feed = { read: [], unread: [] };
 
+		/**
+		 * Initialize the Ian client
```

Comment:
Missing full stop at the end. Update all comments.

Reply:
We follow the [WordPress Coding Standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/javascript/#functions) for all our codes as much as possible.